testing/jormungandr-integration-tests/src/common/jcli/api/key.rs

```diff
@@ -94,13 +94,15 @@ impl Key {
             .stderr(predicates::str::contains(expected_msg_path));
     }
 
+    #[allow(clippy::wrong_self_convention)]
     pub fn to_bytes<S: Into<String>, P: AsRef<Path>>(self, private_key: S, output: P) {
```

Comment:
Here and below, these are not conventional conversion methods. More like `write_to_file`? Cc @dkijania